docs/data/tree-view/overview/overview.md

```diff
@@ -75,7 +75,29 @@ Or increasing the width of the item state indicator to be full-width:
 
 ## Customization
 
-### Custom icons, border and animation
+### Custom icons
+
+You can customize the collapse, expand and end icons of the TreeView.
+
+You can use icon liraries:
+
+{{"demo": "CustomMaterialIcons.js", "defaultCodeOpen": true}}
+
+Or you can create your own components:
+
+{{"demo": "CustomIcons.js", "defaultCodeOpen": false}}
+
+### Adding borders
+
+{{"demo": "BorderedTreeView.js", "defaultCodeOpen": false}}
+
+### Customizing collapse animations
+
```

Comment:
```suggestion
Use the [Transition](/material-ui/transitions/) component from Material UI to add animations for when the Tree View expands and collapses.
```

Reply:
Rephrased a bit, as my approach there is a bit different, but I agree that adding a short explanation is a good idea :)